easybuild/easyconfigs/n/NVHPC/NVHPC-21.5.eb

```diff
@@ -0,0 +1,77 @@
+    {
+        local_tarball_tmpl % 'aarch64':
+            '1a1748cd7cf538199d92ab3b1208935fa4a62708ba21125aeadb328ddc7380d4',
+        local_tarball_tmpl % 'ppc64le':
+            '4674931a5ce28724308cb9cebd546eefa3f0646d3d08adbea28ba5ad27f0c163',
+        local_tarball_tmpl % 'x86_64':
+            '21989e52c58a6914743631c8200de1fec7e10b3449c6c1833f3032ee74b85f8e',
+    }
+]
+
+local_gccver = '10.3.0'
+dependencies = [
+    ('GCCcore', local_gccver),
+    ('binutils', '2.36.1', '', ('GCCcore', local_gccver)),
+    # This is necessary to avoid cases where just libnuma.so.1 is present in the system and -lnuma fails
+    ('numactl', '2.0.14', '', ('GCCcore', local_gccver))
+]
+
+# specify default CUDA version that should be used by NVHPC
+# should match one of the CUDA versions that are included with this NVHPC version
+# (see install_components/Linux_x86_64/21.5/cuda/)
+# for NVHPC 21.5, those are: 11.2, 11.0, 10.2;
+# this version can be tweaked from the EasyBuild command line with
+# --try-amend=default_cuda_version="10.2" (for example)
+default_cuda_version = '11.3'
+
+# NVHPC EasyBlock supports some features, which can be set via CLI or this easyconfig.
+# The following list gives examples for the easyconfig
+#
+# NVHPC needs CUDA to work. Two options are available: 1) Use NVHPC-bundled CUDA, 2) use system CUDA
+# 1) Bundled CUDA
+#    If no easybuild dependency to CUDA is present, the bundled CUDA is taken. A version needs to be specified with
+#      default_cuda_version = "11.0"
+#    in this easyconfig file; alternatively, it can be specified through the command line during installation with
+#      --try-amend=default_cuda_version="10.2"
+# 2) CUDA provided via EasyBuild
+#    Use CUDAcore as a dependency, for example
+#      dependencies = [('CUDAcore', '11.0.2')]
+#    The parameter default_cuda_version still can be set as above.
+#    If not set, it will be deduced from the CUDA module (via $EBVERSIONCUDA)
+#
+# Define a NVHPC-default Compute Capability
+#   cuda_compute_capabilities = "8.0"
+# Can also be specified on the EasyBuild command line via --cuda-compute-capabilities=8.0
+# Only single values supported, not lists of values!
+#
+# Options to add/remove things to/from environment module (defaults shown)
+#   module_byo_compilers = False  # Remove compilers from PATH (Bring-your-own compilers)
+#   module_nvhpc_own_mpi = False  # Add NVHPC's own pre-compiled OpenMPI
+#   module_add_math_libs = False  # Add NVHPC's math libraries (which should be there from CUDA anyway)
+#   module_add_profilers = False  # Add NVHPC's NVIDIA Profilers
+#   module_add_nccl = False       # Add NVHPC's NCCL library
+#   module_add_nvshmem = False    # Add NVHPC's NVSHMEM library
+#   module_add_cuda = False       # Add NVHPC's bundled CUDA
+
+# Temporary patch. Please check if necessary for the new release and remove.
+postinstallcmds = ['sed -i "s/LIBRARY_PATH/LLLIBRARY_PATH/" %(installdir)s/Linux_x86_64/21.5/compilers/bin/siterc']
```

Comment:
This can be removed now, thanks to the fix in https://github.com/easybuilders/easybuild-easyblocks/pull/2453

Reply:
I did remove it as I noticed that but didn't commit.

Reply:
Fixed now!